controller/search_blackbox_test.go

```diff
@@ -934,7 +934,7 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 			workitem.SystemIteration: sprint2.ID.String(),
 		}, s.testIdentity.ID)
 	require.Nil(s.T(), err)
-	s.T().Run("assignee=null after", func(t *testing.T) {
+	s.T().Run("assignee=null after WI creation", func(t *testing.T) {
```

Comment:
I understand that this has nothing to do with the code you wrote but, can we use `testFixture` while creating that one WI object? Or I am okay not making that change here, we can handle it later.

Reply:
Yes, this not a related change.

We will do test refactoring in a separate PR.